### ISSUE TITLE
Abuse of Optional

### ISSUE THREAD

**Issue by CYXK** [apache/shardingsphere-elasticjob#718]
Please answer these questions before submitting your issue. Thanks!
开源不易，我们希望将精力放在完成新功能和解决有价值的问题上，为了让大家的配合更具有效率，请填写以下列出的全部问题

### Which version of Elastic-Job do you using?（您使用的Elastic-Job版本为？）
NA

### Expected behavior （您预期的结果是）
From IDEA Inspection: "Optional was designed to provide a limited mechanism for library method return types where there needed to be a clear way to represent "no result"."

### Actual behavior （实际运行的结果是）
A lot of menthod using optional as param type,this is unreasonable.
I found in source code like this:
```
    /**
     * 作业立刻执行.
     *
     * <p>作业在不与上次运行中作业冲突的情况下才会启动, 并在启动后自动清理此标记.</p>
     *
     * @param jobName 作业名称
     * @param serverIp 作业服务器IP地址
     */
    void trigger(Optional<String> jobName, Optional<String> serverIp);
``` 

### Steps to reproduce the behavior （可重现问题的操作步骤）
NA

### Please provide the reproduce example codes (such as github link)，otherwise we will label the issue as Invalid and close it.（为了节省复现问题的时间，请务必提供可重现的代码，否则我们会将issue直接标记为invalid并关闭）
NA

Code should based on https://github.com/elasticjob/elastic-job-example
（代码请基于 https://github.com/elasticjob/elastic-job-example）

**Comment by compilerduck** [comment]
Hi CYXK, 
Thank you for the suggestion. This seems to be a discussion about how to use "Optional" properly, rather than a real issue. If you search on internet, you will find someone use Optional as return type, and someone use it as function parameter. In my opinion, I prefer to using it as return type. But currently, this is not an important optimization. Anyway, thank you for submitting the issue.

**Comment by terrymanu** [comment]
Java 8 upgrade finished, I can close this issue now.
Fixed by #748
